Controllers/TransactionController.cs

```diff
@@ -24,4 +26,22 @@ public async Task<ActionResult> GetTxByHash([FromRoute] string txHash)
         var result = await Mediator.Send(new GetTransactionByTxHashQuery { TxHash = txHash });
         return Ok(result);
     }
+
+    /// <summary>
+    /// Creates a transaction in the replicator node
+    /// </summary>
+    /// <param name="body">CreateTransactionDto containing transaction ID and payload</param>
+    [HttpPost]
+    [Route("create")]
+    public async Task<ActionResult> CreateTransactionInReplicatorNode([FromBody] CreateTransactionDto body)
+    {
+        var txHash = await Mediator.Send(new CreateTransactionInReplicatorNodeCommand
+        {
+            TransactionId = body.TransactionId,
+            Payload = body.Payload
+        });
+
+        // TODO: How and in what detail do we want to process errors and return them to the API caller?
+        return txHash != null ? Ok(txHash) : Ok("Error when creating transaction in replicator node.");
```

Comment:
we ll go withe the exception approach and a global handler that returns correct http code in case of err